services/parser-database/connector.py

```diff
@@ -20,20 +21,23 @@ def get_documents_to_parse():
 
 
 def get_keywords(text):
-    """Get keywords that relate to this article
+    """Get keywords that relate to this article (Javier's service)
```

Comment:
Javier, exposed. (But actually it would be better to refer to this as the keyword extraction service).

Reply:
again haha, yes, I wen't ahead and changed that